util/featureflag/featureflag.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2021 PingCAP, Inc. Licensed under Apache-2.0.
+
+package featureflag
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/Masterminds/semver"
+	"github.com/gin-gonic/gin"
+
+	"github.com/pingcap/tidb-dashboard/util/rest"
+)
+
+var ErrFeatureUnsupported = rest.ErrForbidden.NewSubtype("feature_unsupported")
```

Comment:
Oh. I'm sorry, I misled you. There is no way to attach the error status code into the error type for now.

Please use `HTTPCodeProperty` when initializing errors:

```go
var ErrFeatureUnsupported = ErrNS.xxxx()

_ = c.Error(ErrFeatureUnsupported.NewWithNoMessage().WithProperty(HTTPCodeProperty(http.StatusForbidden)))
```

or:

```go
_ = c.Error(rest.ErrForbidden.New("<ErrorMessages>")
```